funnel/templates/schedule_subscribe.html.jinja2

```diff
@@ -1,5 +1,5 @@
 <div class="modal__header">
-  <a class="modal__close" href="javascript:void(0);" aria-label="close" rel="modal:close"><i class="material-icons" aria-hidden="true">clear</i></a>
+  <a class="modal__close" href="javascript:void(0);" aria-label="{% trans %}close{% trans %}" rel="modal:close"><i class="material-icons" aria-hidden="true">clear</i></a>
   <h3 class="mui--text-title">{% trans %}Subscribe to the {% endtrans %}{{ project.title }}{% trans %} schedule{% endtrans %}</h3>
```

Comment:
This one needs to be reduced to `Subscribe to the schedule`. Can't insert project title in between a translated string.